assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/TicketPrice.tsx

```diff
@@ -3,11 +3,12 @@ import React, { useCallback } from 'react';
 import { useDataState } from '../data';
 import BaseField from './BaseField';
 import { BaseFieldProps, TicketPriceFieldProps } from './types';
+import TicketPriceField from './TicketPriceField';
 import { useMoneyDisplay } from '@appServices/utilities/money';
 
 type BFP = BaseFieldProps<number>;
 
-const TicketPriceField: React.FC<TicketPriceFieldProps> = (props) => {
+const PriceField: React.FC<TicketPriceFieldProps> = (props) => {
```

Comment:
this was renamed because the `TicketPriceField` wrapper uses `ee-ticket-price-field`